apps/fares/lib/proposed_locations.ex

```diff
@@ -58,7 +58,7 @@ defmodule Fares.ProposedLocations do
     %Location{
       fid: attributes["FID"],
       stop_id: attributes["stop_id"],
-      stop_name: attributes["stop_name"],
+      name: attributes["stop_name"],
```

Comment:
Just curious, why was the name changed?

Reply:
Ah - it wasn't absolutely necessary, so correct me if this was an out-of-scope change.  My reasoning is that since I combined the Proposed Location Card and Retail Location Card into a single Location Card component, I wanted to make the attributes consistent.  So location "name" worked for both, while location "stop_name" only worked for proposed locations.  (But I could see that logic living in the Location Card component instead, if needed.)